File: 백준/Bronze/10810. 공 넣기/공 넣기.cc
#include <stdio.h>

int main()
{
    int basket[101] = {0,};
    int N,M,i,j,k;

    scanf("%d %d", &N, &M);

    for (int a = 0; a < M; a++){
        scanf("%d %d %d", &i, &j, &k);
        for(int b = i; b <= j; b++){
            basket[b] = k;
        }
    }

    for (int c = 1; c <= N; c++){
        printf("%d ", basket[c]);
    }

    return 0;
}

File: 백준/Bronze/10810. 공 넣기/공 넣기.java
import java.util.Scanner;

public class Main
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        int[] basket = new int[101];
        int N = scanner.nextInt();
        int M = scanner.nextInt();
        
        for(int a = 0; a < M; a++)
        {
            int i = scanner.nextInt();
            int j = scanner.nextInt();
            int k = scanner.nextInt();
            for(int b = i; b <= j; b++)
            {
                basket[b] = k; 
            }
        }

        for(int a = 1; a <= N; a++)
        {
            System.out.print(basket[a] + " ");
        }

        scanner.close();
    }
}